### DISCUSSION TITLE
Here's a demo of the Extension Development Workflow using ClojureScript

### DISCUSSION THREAD

**Discussion by PEZ** [microsoft/vscode-discussions#242] [Extension Show and Tell]
I made a video about some of the benefits of using ClojureScript for developing VS Code Extensions. It's a short (9 minutes) demo of me hacking on an extension of mine, trying to relay what it is we LISPers/SmallTalkers/etcetera mean when we speak of Interactive Programming (aka REPL Driven Development). I find it super powerful when hacking on extensions, since restarting the Dev Extension Host can be pretty disruptive.

* https://www.youtube.com/watch?v=fMtvtAfdr90

**Comment by isidorn** [comment]
Great demo, thanks for sharing 👏
